internal/tiltfile/extension/github_fetcher.go

```diff
@@ -0,0 +1,49 @@
+package extension
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"time"
+)
+
+type GithubFetcher struct {
+}
+
+func NewGithubFetcher() *GithubFetcher {
+	return &GithubFetcher{}
+}
+
+const githubTemplate = "https://raw.githubusercontent.com/windmilleng/tilt-extensions/master/%s/Tiltfile"
+
+func (f *GithubFetcher) Fetch(ctx context.Context, moduleName string) (ModuleContents, error) {
+	c := &http.Client{
+		Timeout: 20 * time.Second,
+	}
+
+	urlText := fmt.Sprintf(githubTemplate, url.PathEscape(moduleName))
```

Comment:
should there be some stricter validation for what is a valid module name?

Someone is always going to put something weird in there (like a docker image name+tag), and get confused why it doesn't work